### PR TITLE
WIP feat: introducing GPU services

### PR DESCRIPTION
**Pull request by tillt**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-76761

TODO:
- insert list of support distros for phase 1
- insert link to konvoy-image-builder docs
- insert link to on-prem GPU installation guide
- insert guide on adding SecurityGroup setup for GPU metrics; open port 9400 for access by other workers

Questions:
- How can we disable the Nvidia platform service on Kommander2? -> check Max's demo for an answer!
- How to deploy a specific/later version of this service on Kommander2?

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.0/gpu/index.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+```bash
+kubectl label nodes <NODE> konvoy.mesosphere.com/gpu-provider=NVIDIA
+```
+
+Then when installing Kommander, enable the `nvidia` services. 
+
+```bash
+helm install -n kommander --create-namespace kommander-bootstrap kommander/kommander-bootstrap --set services.nvidia.enabled=true
+```
```
Comment (tillt) [inline pages/dkp/kommander/2.0/gpu/index.md:76]: This is the essential

File: pages/dkp/kommander/2.0/gpu/index.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+title: GPUs
+menuWeight: 150
+excerpt: Configure GPU for Kommander cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+This section describes the Nvidia GPU support on Kommander. This document assumes familiarity with Kubernetes GPU support. AMD GPUs are currently not supported.
```
Comment (jongiddy) [inline pages/dkp/kommander/2.0/gpu/index.md:13]: Why do we specifically name AMD GPUs here? Would it be more general to say `Currently Kommander only supports Nvidia GPUs.`

File: pages/dkp/kommander/2.0/gpu/index.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+# Kommander GPU Overview
+
+GPU support on Kommander is achieved by using the [nvidia-container-runtime][nvidia_container_runtime].
+With the Nvidia GPU turned on, Kommander configures the container runtime for running GPU containers, and installs all the necessary items to power up the Nvidia GPU devices.
+Kommander runs every Nvidia GPU dependent component as daemonsets, making it easier to manage and upgrade.
```
Comment (jongiddy) [inline pages/dkp/kommander/2.0/gpu/index.md:19]: Perhaps move this sentence below the list of components. That would also make it more natural to say `Kommander runs these components as daemonsets,...`

File: pages/dkp/kommander/2.0/gpu/index.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+|=============================================================================|
+|  No running processes found                                                 |
++-----------------------------------------------------------------------------+
+```
+
+## Configuration
+
+### Enable Nvidia GPU on Kommander
+
+To enable Nvidia GPU support on Kommander the first thing to do is adding the label `konvoy.mesosphere.com/gpu-provider=NVIDIA` to each GPU node with installed Nvidia host components.
```
Comment (jongiddy) [inline pages/dkp/kommander/2.0/gpu/index.md:66]:
```suggestion
To enable Nvidia GPU support on Kommander add the label `konvoy.mesosphere.com/gpu-provider=NVIDIA` to each GPU node with installed Nvidia host components.
```

**Comment by cmastr** [comment]
Kommander 2 docs should originate in the Kommander repo please:
https://github.com/mesosphere/kommander/tree/main/docs/site

**Comment by cmastr** [comment]
https://github.com/mesosphere/kommander/pull/479

replacement

**Pull request closed without merging**